### ISSUE TITLE
Ability to fetch data by member id

### ISSUE THREAD

**Issue by chrisguindon** [EclipseFdn/cbi-sponsorships-api#24]
On the membership portal, we display `resource packs`, `dedicated agents` and `GitHub large runners`. However, to do so, the client must fetch the full response of https://api.eclipse.org/cbi/sponsorships/.

If would be more efficient if we could query the API to only fetch data for a specific organization using the org id:

https://api.eclipse.org/cbi/sponsorships/members/757

This is related to https://github.com/EclipseFdn/cbi-sponsorships-api/issues/23

**Comment by chrisguindon** [comment]
Closing this issue. We took a closer look at the source code for this project and I won't think this is something we can do with this API.

I think my team can manage this by exposing this data via our existing API.

TLDR; We will fetch a copy of https://api.eclipse.org/cbi/sponsorships/ once a day and expose the data we membership and project API.
